cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -127,20 +132,39 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		}
 	} else if !isWCOW {
```

Comment:
would a job container be isWCOW? Should we move this below the section for job containers?

Reply:
`isWCOW` will be true as we get the same spec, but we wouldn't fall into the first case as it's not isolated, and we wouldn't fall in this case either because of the above. I do think moving it in the middle makes more sense though, feels better to have the error case as the last.